src/app/(site)/movies/[movieId]/page.tsx

```diff
@@ -1,39 +1,105 @@
-import React from "react";
-import { notFound } from "next/navigation";
+'use client';
```

Comment:
The formatting has changed in this file, which is making it more difficult to understand what has actually, _substantially_ changed, because it looks like all lines have changed.

Avoid making formatting changes. If anyone uses automatic formatting, make sure the entire team uses the same settings. This can be achieved in different ways, e.g. using a formatter like `prettier` or using and `.editorconfig` file.